lib/atsign_exists.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:at_client/at_client.dart';
+
+/// Checks if the provided atSign's atServer has been properly activated with a public RSA key.
+/// `atClient` must be authenticated
+/// `atSign` is the atSign to check
+Future<bool> atSignIsActivated(final AtClient atClient, String atSign) async {
```

Comment:
Can you add into the sshnp_utils.dart please? Keeps things tidy

Reply:
https://github.com/atsign-foundation/sshnoports/pull/211/commits/51781b9e6ac9b2097660647043776aa23ea38789